### ISSUE TITLE
When VR will be supported ?

### ISSUE THREAD

**Issue by shantee** [tentone/nunuStudio#415]
Hello

Do you have an ETA about the VR bug ? 
I tried to run different nunuStudio program (scenes) on my Occulus Quest but it doesn't work (the VR button doesn't even display.
Also what kind of headset is supported for the moment ?

**Comment by tentone** [comment]
Hello

Currently i am finishing the migration to modules of the whole project, that is pretty much ready.

The problem with VR will be tackled right after to figure out why we are getting a black screen in VR mode.

All WebXR compatible browsers/headsets should be working after i fix the bug.

Thanks a lot once again, Sorry for this big delay.

**Comment by tentone** [comment]
Hello

The VR issues have been solver in the new version 0.98.12 available on the web.

Thanks a lot for your patience.

Cheers

**Comment by shantee** [comment]
Hello , sorry  but no it still doesn't work , at least not on my occulus quest.
I have been trying the stock browser and also firefox.
The app loads correctly but I can't enter into VR mode (the vr icon doesn't appear) 

I tried :
Making a simple scene using the web editor
Making a simple scene using the latest github dev version.
And finally with the VR exemple (in learn -> basic ...)

If you need more info don't hesitate

**Comment by tentone** [comment]
Hello

Have you enabled VR in the program configuration? Does the icon appear in this demo https://www.nunustudio.org/page/examples/demo.html?nsp=files/littlevoxelisland.nsp?

The editor has been updated and as far I have tested seems to be working properly with WebXR compatible browsers.

Thanks a lot

**Comment by shantee** [comment]
Yes I did check that...That is so weird! When I go to your online example (the little voxel island) with my occulus quest it is working fine ! even on chrome on my desktop computer the "VR" icon is displayed correctly.

So I decided  to transfer your example , the whole folder,  to some server i have....But same problem it doesn't work. 
Look everything is the same I just copied / pasted  : http://play.zfx.fr/testvr/demo.html?nsp=files/littlevoxelisland.nsp
